features/situation-feedback/elastic/src/main/java/org/opennms/features/situationfeedback/elastic/ElasticFeedbackRepository.java

```diff
@@ -125,6 +127,42 @@ public void persist(List<AlarmFeedback> feedback) throws FeedbackException {
         return search(query);
     }
 
+    @Override
+    public List<String> getTags(String prefix) throws FeedbackException {
+        String query = "{\n" + 
+                "  \"size\": 0,\n" + 
+                "  \"aggs\": {\n" + 
+                "    \"terms\": {\n" + 
+                "      \"terms\": {\n" + 
+                "        \"field\": \"tags\",\n" + 
+                "        \"include\": \"" + prefix + ".*\"\n" + 
```

Comment:
Can we ensure that the prefix is properly escaped? Looks like the query will break if it contains a `"` now.

Reply:
now using ```gson.toJson()``` to escape 'prefix'